### PR TITLE
ROX-12712: Address scanner OOMs in OpenShift e2e tests

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

As per https://github.com/stackrox/stackrox/pull/751, don't alter scanner requests/limits in CI. This PR makes the same change for helm based installs which is used in openshift e2e tests.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

CI is sufficient

### CODE REVIEW

**Comment by gavin-stackrox** [comment]
This is new, when trying to dump the DB when tests fail:
```
    go: github.com/stackrox/scanner@v0.0.0-20210602003504-057b882c8d87 requires
    	github.com/stackrox/rox@v0.0.0-20201222223839-042406f3664c: invalid version: git ls-remote -q origin in /go/pkg/mod/cache/vcs/490949d4ae5eefdc73aceb001a97037702f10305266d6ab409726415cbd222ed: exit status 128:
    	fatal: could not read Username for 'https://github.com': terminal prompts disabled
    Confirm the import path was entered correctly.
    If this is a private repository, see https://golang.org/doc/faq#git_https for additional information.
```
probably not related to the changes on the PR though.

**Comment by gavin-stackrox** [comment]
The OpenShift 4 tests fail because sensor is pending. Presumably because resources are starved due to the scanner changes. 
```
sensor-54f6c5874d-b2gn7             0/1     Pending            0               9m28s   <none>        <none>                         <none>           <none>
```

File: tests/upgrade/run.sh [hunk 1]
```diff
@@ -371,8 +371,8 @@ test_upgrade_paths() {
 
     local log_output_dir="$1"
 
-    EARLIER_SHA="9f82d2713cfec4b5c876d8dc0149f6d9cd70d349"
-    EARLIER_TAG="3.63.x-163-g2c4fe1563c"
+    EARLIER_SHA="6cda3d46cdf9be19389e313f2f112c031dcd0172"
+    EARLIER_TAG="3.70.2-1-gb71c18b7c8"
```
Comment (RTann) [inline tests/upgrade/run.sh:375]: Can we just do 3.70.2?

Comment (gavin-stackrox) [reply tests/upgrade/run.sh:375]: > Can we just do 3.70.2?

yeah, I suppose we could

**Comment by gavin-stackrox** [comment]
ci/prow/gke-upgrade-tests

**Comment by gavin-stackrox** [comment]
@RTann I've updated this PR so that it is only addressing the issue with OpenShift e2e test failures (the original reason behind ROX-12712).

**Review by c-du — APPROVED** [review]

**Pull request merged**